### PR TITLE
Fix up Vala support

### PR DESCRIPTION
**Pull request by lw64**

### CODE REVIEW

File: src/Library/demos/Welcome/main.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Gtk from "gi://Gtk?version=4.0";
 
 Gtk.init();
+Adw.init();
```
Comment (sonnyp) [inline src/Library/demos/Welcome/main.js:4]: `Adw.init()` already calls `Gtk.init()` so it should be one or the other.

This particular demo doesn't use `Adw`

Comment (lw64) [reply src/Library/demos/Welcome/main.js:4]: That is right. But the style is different then.

Comment (sonnyp) [reply src/Library/demos/Welcome/main.js]: Should we always use `Adw.init()` ?

**Comment by sonnyp** [comment]
https://github.com/sonnyp/Workbench/commit/4d029ab2d65d1aca4b234d946c659ef16564ede6

**Pull request closed without merging**